core/templates/dev/head/pages/story-viewer-page/story-viewer-page.directive.ts

```diff
@@ -250,6 +271,17 @@ angular.module('oppia').directive('storyViewerPage', [
                   storyDataDict.story_title + ' - Oppia');
                 ctrl.storyTitle = storyDataDict.story_title;
                 ctrl.storyDescription = storyDataDict.story_description;
+
+                // Taking the characteristics of the first chapter as
+                // placeholder to be displayed on the right, since all
+                // explorations in a chapter should have the same category, and
```

Comment:
I thought "one chapter" == "one exploration". Is this comment wrong?

Reply:
I meant all explorations across all chapters, will change it.

Reply:
Done